extension-requests/local-utils.js

```diff
@@ -62,6 +62,29 @@ const generateExtensionRequestParams = (nextPageParams) => {
   const uri = `/extension-requests?${urlSearchParams.toString()}`;
   return uri;
 };
+
+const parseExtensionRequestParams = (uri, nextPageParamsObject) => {
+  const urlSearchParams = new URLSearchParams(uri);
+
+  for (const [key, value] of urlSearchParams.entries()) {
+    if (key === 'q') {
+      const searchQueries = value.split(',');
+      searchQueries.forEach((query) => {
+        if (!query) return;
+        const [queryKey, queryValue] = query.split(':');
+        if (queryValue?.includes('+')) {
+          nextPageParamsObject[queryKey] = queryValue.split('+');
+        } else if (queryValue) {
+          nextPageParamsObject[queryKey] = queryValue;
+        }
+      });
+    } else {
+      nextPageParamsObject[key] = value;
+    }
+  }
+  return nextPageParamsObject;
+};
```

Comment:
Please provide an example for which type of URI it is parsing

Reply:
Done